src/test/components/__snapshots__/MarkerTooltipContents.test.js.snap

```diff
@@ -1352,17 +1345,17 @@ Array [
       <div
         className="tooltipLabel"
       >
-        pri
+        Priority
         :
       </div>
       -20
       <div
         className="tooltipLabel"
       >
-        count
+        Requested bytes
         :
       </div>
-      0
+      0.000B
```

Comment:
Optional follow-up: `0.00B` seems like a weird value to show. I would prefer `0B`. Also, do we know if bytes will ever be fractional? It seems like it will always be an integer value. Here is the culprit:

https://github.com/devtools-html/perf.html/blob/f77a9c17e9adac5ee0be624243d63b50243d2ffc/src/utils/format-numbers.js#L57

Reply:
I was wondering, if this will ever show up in the tooltip as a value? 
If not, would it make sense to change it?

> Also, do we know if bytes will ever be fractional?
It should always be an integer here, but I am not sure if this behavior is consistent.